Apps/WebClient/src/ClientApp/src/components/report/ReportsComponent.vue

```diff
@@ -575,6 +588,7 @@ export default class ReportsComponent extends Vue {
                 ref="report"
                 :hdid="hdid"
                 :filter="reportFilter"
+                :is-dependent="isDependent"
```

Comment:
The parent \<div\> should not have the "preview" class added for dependent reports, which don't have previews.  It's adding scrollbars & unnecessarily increasing the height of the tab.

Line 584:

```
class="preview"
```

Should be:

```
:class="{ preview: !isDependent }"
```